src/fastga/models/aerodynamics/components/wing/compute_cl_yaw_rate_wing.py

```diff
@@ -35,72 +37,233 @@ class ComputeClYawRateWing(FigureDigitization):
 
+    # pylint: disable=missing-function-docstring
+    # Overriding OpenMDAO setup
     def setup(self):
-        self.add_input("data:geometry:wing:aspect_ratio", val=np.nan)
-        self.add_input("data:geometry:wing:taper_ratio", val=np.nan)
-        self.add_input("data:geometry:wing:sweep_25", val=np.nan, units="rad")
-        self.add_input("data:geometry:wing:dihedral", val=np.nan, units="deg")
+        ls_tag = "low_speed" if self.options["low_speed_aero"] else "cruise"
+
+        self.add_subsystem(
+            name="b_coeff_" + ls_tag,
+            subsys=_BCoeff(low_speed_aero=self.options["low_speed_aero"]),
+            promotes=["data:*"],
+        )
+        self.add_subsystem(
+            name="mach_correction_" + ls_tag, subsys=_MachCorrection(), promotes=["data:*"]
+        )
+        self.add_subsystem(
+            name="dihedral_effect_" + ls_tag,
+            subsys=_ClRollMomentDihedralEffect(),
+            promotes=["data:*"],
+        )
+        self.add_subsystem(
+            name="twist_effect_" + ls_tag,
+            subsys=_ClRollMomentFromTwist(),
+            promotes=["data:*"],
+        )
+        self.add_subsystem(
+            name="lift_effect_part_a_" + ls_tag,
+            subsys=_ClRollMomentLiftEffectPartA(),
+            promotes=["data:*"],
+        )
+        self.add_subsystem(
+            name="lift_effect_part_b_" + ls_tag,
+            subsys=_ClRollMomentLiftEffectPartB(),
+            promotes=["data:*"],
+        )
+        self.add_subsystem(
+            name="Cl_w_" + ls_tag,
+            subsys=_Clw(low_speed_aero=self.options["low_speed_aero"]),
+            promotes=["data:*", "settings:*"],
+        )
+        self.add_subsystem(
+            name="Cl_r_wing_" + ls_tag,
+            subsys=_ClrWing(low_speed_aero=self.options["low_speed_aero"]),
+            promotes=["data:*"],
+        )
+
+        self.connect(
+            "b_coeff_" + ls_tag + ".b_coeff_mach",
+            "mach_correction_" + ls_tag + ".b_coeff_mach",
+        )
+        self.connect(
+            "lift_effect_part_a_" + ls_tag + ".k_coefficient",
+            "lift_effect_part_b_" + ls_tag + ".k_coefficient",
+        )
+        self.connect(
+            "mach_correction_" + ls_tag + ".mach_correction",
+            "Cl_r_wing_" + ls_tag + ".mach_correction",
+        )
+        self.connect(
+            "dihedral_effect_" + ls_tag + ".dihedral_effect",
+            "Cl_r_wing_" + ls_tag + ".dihedral_effect",
+        )
+        self.connect(
+            "twist_effect_" + ls_tag + ".cl_r_twist_effect",
+            "Cl_r_wing_" + ls_tag + ".cl_r_twist_effect",
+        )
+        self.connect(
+            "lift_effect_part_b_" + ls_tag + ".lift_effect_mach_0",
+            "Cl_r_wing_" + ls_tag + ".lift_effect_mach_0",
+        )
+        self.connect(
+            "Cl_w_" + ls_tag + ".cl_w",
+            "Cl_r_wing_" + ls_tag + ".cl_w",
+        )
+
+
+class _Clw(om.ExplicitComponent):
```

Comment:
Doesn't this one already exist and coud be reuse ? Even if no, I remember it is used quite often, so maybe, just like for the beta coefficient is could be placed somewhere accessible by the other components,